src/Common/tests/System/Collections/TestingTypes.cs

```diff
@@ -147,6 +147,9 @@ public int GetHashCode(int x)
 
     #region TestClasses
 
+#if netstandard17
```

Comment:
Do you think the #if around Serializble is worth it? Over just providing a stub SerializableAttribute we provide for non-netstandard17?